canonical-web-and-design/hiring.canonical.com
# Add meet and greet charts

## Done
Meet and greet interview stage's charts are added

## QA
- Run: `dotrun`
- Open `http://0.0.0.0:8501/stages/meet-and-greet-interview`
- Verify charts are displayed correctly.

## Issue
Fixes #186
